crates/steel-parser/src/lexer.rs

```diff
@@ -308,6 +309,7 @@ impl<'a> Lexer<'a> {
             NEG_INFINITY => TokenType::Number(RealLiteral::Float(f64::NEG_INFINITY).into()),
             NAN => TokenType::Number(RealLiteral::Float(f64::NAN).into()),
             NEG_NAN => TokenType::Number(RealLiteral::Float(f64::NAN).into()),
+            "|.|" => TokenType::Identifier("."),
```

Comment:
I'm hardcoding this escaped identifier for now so I can fix `stack_struct.scm`

Reply:
For what its worth we could change `stack_struct.scm` - its just a test I've had around for a long time. I can see how it behaves on racket before we do anything though